examples/petstore/controllers/pets.go

```diff
@@ -126,6 +130,23 @@ func (rs PetsResources) filterPets(c fuego.ContextNoBody) ([]models.Pets, error)
 	})
 }
 
+type FilterParams struct {
+	Name        string `query:"name" description:"Filter by name" example:"cat name"`
+	YoungerThan int    `query:"younger_than"`
```

Comment:
Could we sneak an array in here for golden diff purposes?

Reply:
Well spotted indeed, we're now declaring them in the spec! Before, it was unavailable on the spec but deserializable